scripts/run-ft-benchmark.py

```diff
@@ -0,0 +1,101 @@
+import argparse
+import os
+import subprocess
+
+LOCK_FILE = "~/tmp/run-ft-benchmark.lock"
+REPO_DIR = "~/nearcore"
+
+
+def create_lock_file(user: str) -> None:
+    if os.path.exists(LOCK_FILE):
+        with open(LOCK_FILE, 'r') as f:
+            running_user = f.read().strip()
+        raise RuntimeError(f"{running_user} already running benchmark")
+    with open(LOCK_FILE, 'w') as f:
+        f.write(user)
+
+
+def remove_lock_file() -> None:
+    if os.path.exists(LOCK_FILE):
+        os.remove(LOCK_FILE)
+    else:
+        raise RuntimeError("Somebody already removed the lock file!!!")
+
+
+def checkout_repo(commit: str = None) -> None:
```

Comment:
I think it should be the responsibility of the user, to check out the correct version of the code before running the benchmark, so I would remove this functionality from the script